src/giftCards/GiftCardCreateDialog/GiftCardCreateDialogContent.tsx

```diff
@@ -76,8 +76,12 @@ const GiftCardCreateDialogContent: React.FC<GiftCardCreateDialogContentProps> =
   };
 
   return (
-    <>
-      <DialogTitle disableTypography>{intl.formatMessage(messages.title)}</DialogTitle>
+    <DashboardModal.Content
+      __maxWidth={DASHBOARD_MODAL_WIDTH}
```

Comment:
issue: if this is the `DashbaordModal` then why `DASHBOARD_MODAL_WIDTH` is not its default width/maxWidth, instead you import it all over the files? This is a bit weird design

Reply:
because I didn't want to expand the scope of this task and check all existing models. If we want to unify this in some way, we can create a task for it and do this in a separate task.